File: Day 2/triangleArea.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main()
{
    float base, height;

    cout << "Type your triangle base:" << endl;
    cin>> base;

    cout << "Type your triangle height:" << endl;
    cin>> height;

    float area = base*height/2;
    cout << "Your triangle area is "<< area << endl;

    float side1, side2, side3;
    float x;

    side1 = base;

    cout << "Type the lenght of one side:" << endl;
    cin>> side2;

    x = side1 - sqrt( pow(side2, 2) - pow(height, 2) );
    // cout << "x = "  << x << endl;

    side3 = sqrt( pow(height, 2) + pow( x, 2) );
    // cout << side3 << endl;

    if(side1 == side2 && side2 == side3)
        cout <<"It is equilateral" << endl;
    else if(side1 != side2 && side2 != side3 && side3 != side1)
        cout <<"It is scalene" << endl;
    else
        cout <<"It is isosceles" << endl;


    return 0;
}

File: Day 2/triangleArea.js
const Triangle = {
    base: 30,
    height: 25,
    area: 0,
    type: ' '
}

function calcTriangleArea(Triangle){
    Triangle.area = Triangle.base * Triangle.height /2
}

function triangleType(Triangle){
    //Another triangle data is needed
    side1 = Triangle.base
    side2 = 30
    x = side1 - Math.sqrt( Math.pow(side2, 2) - Math.pow(Triangle.height, 2) )
    side3 = Math.sqrt( Math.pow(Triangle.height, 2) + Math.pow( x, 2) )

    if(side1 === side2 && side2 === side3){
        Triangle.type = 'Equilateral'
    } else if(side1 !== side2 && side2 !== side3 && side3 !== side1){
        Triangle.type = 'Scalene'
    } else {
        Triangle.type = 'Isosceles'
    }

}

calcTriangleArea(Triangle)
triangleType(Triangle)

console.log(Triangle)